src/host/PtySignalInputThread.cpp

```diff
@@ -72,7 +72,7 @@ void PtySignalInputThread::ConnectConsole() noexcept
     }
     if (_initialShowHide)
     {
-        _DoShowHide(_initialShowHide->show);
+        _DoShowHide(*_initialShowHide);

```

Comment:
(for my education) why is `ShowHideData` a struct? Why not just make `_initialShowHide` an `optional<unsigned short>` directly?

Reply:
I suppose it was done to make the code more consistent? I'd probably would've done it the same way.

Reply:
FYI We made everything structs so that they could be read directly "off the wire".